src/NuGet.Clients/NuGet.PackageManagement.UI/UserInterfaceService/SolutionUserOptions.cs

```diff
@@ -6,19 +6,18 @@
 using Microsoft.VisualStudio;
 using Microsoft.VisualStudio.Shell;
 using Microsoft.VisualStudio.Shell.Interop;
-using NuGet.PackageManagement.UI;
 using NuGet.VisualStudio;
 using IStream = Microsoft.VisualStudio.OLE.Interop.IStream;
 
-namespace NuGetVSExtension
+namespace NuGet.PackageManagement.UI
 {
     /// <summary>
     /// Persists user settings in a solution user options file (*.suo).
     /// </summary>
     [Export]
     [Export(typeof(IUserSettingsManager))]
     [PartCreationPolicy(CreationPolicy.Shared)]
-    internal sealed class SolutionUserOptions : IUserSettingsManager, IVsPersistSolutionOpts
+    public sealed class SolutionUserOptions : IUserSettingsManager, IVsPersistSolutionOpts
```

Comment:
Why are we making it public?

Reply:
cause it is in in PM UI now...and we import it from nuget.tools, not via the interface, but via the type.
we need some methods from both interfaces.

Reply:
It's very weird that we are using the type :D